well-architected/cross-cutting-guides/implementing-recommendations.md

```diff
@@ -30,7 +30,7 @@ Assess your workload by completing the assessment. The recommendations for your
 
 ## Prioritize and implement recommendations
 
-Workload owners and key stakeholders should prioritize the recommendations in accordance with the team's standard work prioritization process, factoring in the applicability of the recommendations and any tradeoffs associated with a specific design decision. For example, recommendations might be assigned to a specific owner, or a recommendation might be postponed or dismissed. Like all planned work, the recommendation should be tracked until it's resolved, as part of the workload's SDLC.
+Before initiating the assessment, it is essential to prioritize the pillars according to your specific business needs. This approach will allow you to allocate your efforts more effectively and efficiently. Workload owners and key stakeholders should prioritize the recommendations in accordance with the team's standard work prioritization process, factoring in the applicability of the recommendations and any tradeoffs associated with a specific design decision. For example, recommendations might be assigned to a specific owner, or a recommendation might be postponed or dismissed. Like all planned work, the recommendation should be tracked until it's resolved, as part of the workload's SDLC. It is important to acknowledge that every decision entails trade-offs. Therefore, focusing on the rationale behind your decision-making process is more critical than getting bogged down in implementation details.
```

Comment:
> Therefore, focusing on the rationale behind your decision-making process is more critical than getting bogged down in implementation details.

Why are those the only options?  I feel like this is a strawman. You can have both rational and crisp implementation details -- matter if fact, I would hope everyone does.  What are you trying to teach here?  Can you say it another way that doesn't sound like you're pre-scolding the reader for something they haven't done yet?

Reply:
Well, I just tried to make it fancier but seems I failed. Basically I wanted to say ""why" is more important than "how"". I think I should borrow following sentence from [cloud design patterns](https://learn.microsoft.com/en-us/azure/architecture/patterns/). Earlier we refined it
`
You need to pay attention more to why you're choosing a certain pattern than to how to implement it.`